frontend/components/Calendar/EventEditor.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { useContext, useEffect, useState } from 'react';
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
Is this necessary?

Reply:
Well I'm  sure we can attempt to build it away, I gave it like 10 minutes and then I gave up. You are very welcome to give it a try!